content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -1,31 +1,43 @@
 ---
-title: Multi-network Mesh Expansion
-description: Integrate VMs and bare metal hosts into an Istio mesh deployed on Kubernetes with gateways.
+title: Virtual Machines in Multi-network Meshes
+description: Learn how to add a service running on a virtual machine to your multi-network
```

Comment:
in/on

Reply:
I'm doing `on a virtual machine` throughout the content.